### PR TITLE
Feature/danh vt/t02 display the first 100 jobs

### PR DESCRIPTION
**Pull request by vutiendanhvgu01**
https://deployt02color.surge.sh/

### CODE REVIEW

File: src/containers/Home/index.tsx [hunk 1]
```diff
@@ -40,6 +40,17 @@ function Home() {
     <div className="App">
       <h1 style={{ textAlign: "left" }}>Nhà tuyển dụng hàng đầu</h1>
       <h2>{count} Có Việc Làm IT Cho Developer "Chất"</h2>
+      <div style={{ display: "flex", flexWrap: "wrap" }}>
+        {job.map((j) => {
```
Comment (Tiencmfaba) [inline src/containers/Home/index.tsx:44]: should be `jobs.map((job)`

Comment (vutiendanhvgu01) [reply src/containers/Home/index.tsx]: I have already fixed

File: src/containers/Home/index.tsx [hunk 2]
```diff
@@ -40,6 +41,22 @@ function Home() {
     <div className="App">
       <h1 style={{ textAlign: "left" }}>Nhà tuyển dụng hàng đầu</h1>
       <h2>{count} Có Việc Làm IT Cho Developer "Chất"</h2>
+      <div
+        style={{ display: "flex", flexWrap: "wrap", justifyContent: "center" }}
```
Comment (Tiencmfaba) [inline src/containers/Home/index.tsx:45]: use a class and styles for this

File: src/containers/Home/styles.scss [hunk 3]
```diff
@@ -41,14 +41,36 @@
...
+      .content-tag{
+        border-radius: 30px;
+        position: absolute;
+        display: inline-block;
+        width: 100px;
+        height: 100px;
+        z-index: 100;
+        right: 0;
+        top:0;
+        span{
```
Comment (Tiencmfaba) [inline src/containers/Home/styles.scss:61]: why you define content__status classes, but i cannot see using instead of span tag ??

Comment (vutiendanhvgu01) [reply src/containers/Home/styles.scss]: I have already fixed

File: src/containers/Home/styles.scss [hunk 4]
```diff
@@ -34,23 +34,82 @@
...
+      .content-description{
+        position: absolute;
+        top: 220px;
+      }
+      .content__image{
+        position: absolute;
+        width: 100%;
+        height: 200px;
+        bottom: 100px;
+        .content__image-random{
```
Comment (Tiencmfaba) [inline src/containers/Home/styles.scss:76]: &-random

File: src/containers/Home/index.tsx [hunk 5]
```diff
@@ -15,31 +17,81 @@ function Home() {
...
+        Nhà tuyển dụng hàng đầu
+      </h1>
       <h2>{count} Có Việc Làm IT Cho Developer "Chất"</h2>
+      <LoadingOverlay
+        active={loadingContent()}
+        spinner={<BounceLoader className="loading" />}
+        text="Loading your content..."
+      >
+        <div className="job__list" data-testid="loading">
+          {jobs.map((job) => {
```
Comment (Tiencmfaba) [inline src/containers/Home/index.tsx:55]: add type for job

**Comment by Tiencmfaba** [comment]
remove comment

**Pull request merged**